proto/observer/params.proto

```diff
@@ -56,4 +56,5 @@ message Params {
   option (gogoproto.goproto_stringer) = false;
   repeated ObserverParams observer_params =1;
   repeated Admin_Policy admin_policy= 2;
+  int64 ballot_maturity_blocks = 3;
```

Comment:
I was curious what this maturity was for? Does the ballot have to be finalized for a certain number of blocks?